css-scroll-snap-2/Overview.bs

```diff
@@ -131,15 +131,19 @@ Motivating Examples {#examples}
 Setting Where Scroll Starts {#properties-on-the-scroll-container}
 =================================================================
 
-The 'scroll-start-target' property {#scroll-start-target}
+The 'scroll-initial-target' property {#scroll-initial-target}
 -------------------------------------------
 
+Issue: The name "scroll-initial-target" is chosen as the least worst option
+which expresses that it should be scrolled to when its scroll container
+first shows up on the page. A better name for the property would be very much welcome.
```

Comment:
It would be helpful to provide some indication to the reader what some of the concerns and constraints are, or at least link to the issue so that people can easily find out what the concerns were about this name, e.g. https://drafts.csswg.org/scroll-animations-1/#issue-e4fe9011 or https://drafts.csswg.org/static/css-display-4/#issue-0ae46664

Reply:
Good idea. Done.